themes/theme-ios11/themeApi/ProductCard/components/Render/index.jsx

```diff
@@ -11,6 +11,7 @@ import {
 } from '@shopgate/engage/product';
 import Link from '@shopgate/pwa-common/components/Link';
 import RatingStars from '@shopgate/pwa-ui-shared/RatingStars';
+import { useWidgetSettings } from '../../../../../../libraries/engage/core/hooks/useWidgetSettings';
```

Comment:
please use '@shopgate/engage/core' to import 'useWidgetSettings', because there's an index file which exports among other things all hooks.